roles/openshift_node/tasks/main.yml

```diff
@@ -73,3 +73,24 @@
 
 - name: Start and enable openshift-node
   service: name=openshift-node enabled=yes state=started
+
+- name: Check scheduleable state
+  delegate_to: "{{ openshift_first_master }}"
+  command: >
+    {{ openshift.common.client_binary }} get node {{ inventory_hostname }}
+  register: get_node
```

Comment:
Role variables need to have a prefix based off the role name per our [style guide](https://github.com/openshift/openshift-ansible/blob/master/docs/style_guide.adoc#ansible-role-variables)

I suggest:
```
register: ond_get_node
```